ACK https://github.com/bitcoin/bitcoin/pull/8070/commits/f4119c6c988ea24a5218aa6bc67e57e47e051547

To reproduce the problem without this change:

```
for i in {1..10000}; do echo $i; ./test_bitcoin --run_test=net_tests || break; done
...
Running 2 test cases...
test/net_tests.cpp:92: error in "caddrdb_read": check addrman1.size() == 3 failed
test/net_tests.cpp:102: error in "caddrdb_read": check addrman2.size() == 3 failed

*** 2 failures detected in test suite "Bitcoin Test Suite"
```

When you are here, please fix s/adddrman/addrman/ here: https://github.com/bitcoin/bitcoin/pull/8070/commits/f4119c6c988ea24a5218aa6bc67e57e47e051547#diff-03fc2f333374fd229824451793c40462R131

--- a/src/test/net_tests.cpp
+++ b/src/test/net_tests.cpp
@@ -128,7 +128,7 @@ BOOST_AUTO_TEST_CASE(caddrdb_read_corrupted)
     } catch (const std::exception& e) {
         exceptionThrown = true;
     }
-    // Even through de-serialization failed adddrman is not left in a clean state.
+    // Even through de-serialization failed addrman is not left in a clean state.
     BOOST_CHECK(addrman1.size() == 1);
     BOOST_CHECK(exceptionThrown);
 

